src/tiled/commandbutton.cpp

```diff
@@ -34,6 +35,7 @@ using namespace Tiled::Internal;
 CommandButton::CommandButton(QWidget *parent)
     : QToolButton(parent)
     , mMenu(new QMenu(this))
+    , mCommandManager(new CommandManager(this))
```

Comment:
You don't pass around singletons! Please look up the design pattern. See the `TilesetManager` for an example singleton.